src/executePrettier/buildPrettierOptions.js

```diff
@@ -70,8 +72,10 @@ const buildPrettierOptions = (editor: TextEditor) => {
   return {
     ...optionsFromSettings,
     ...buildEditorConfigOptionsIfAppropriate(editor),
-    ...getPrettierConfigOptions(editor),
   };
 };
 
+const buildPrettierOptions = (editor: TextEditor) =>
+  getPrettierConfigOptions(editor) || getOptionsFromSettings(editor);
```

Comment:
Does this mean we ignore the user's parser configuration settings if there is a prettier config found? That might be confusing as those settings exist outside the "prettier defaults" section and don't have any indication that they will be ignored.

Reply:
True, we only want to discard the prettier configuration settings explicitly, not every setting that `prettier-atom` has. In some advanced cases, a user may want to select a custom parser. We shouldn't take those things away, simply merge over them.

Reply:
I would suggest moving the Prettier formatting settings into their own block to be ignored if a configuration was already detected and then merge in the parser settings all the time as Prettier should be told which language it is working against, which we know from the editor. :D

Reply:
Sounds good.  Sorry for overlooking that.